builder/install/50-luajit.sh

```diff
@@ -2,8 +2,14 @@
 
 set -e
 
-cd third_party/luajit
-
-cp COPYRIGHT "${LICENSE_DIR}/LuaJIT-${LUAJIT_VERSION}"
+if [ "$(arch)" = "x86_64" ]; then
+    cd third_party/luajit
+    cp COPYRIGHT "${LICENSE_DIR}/LuaJIT-${LUAJIT_VERSION}"
+else
+    cd third_party
+    git clone https://github.com/openresty/luajit2.git
+    cd luajit2
+    git checkout v"${LUAJIT2_VERSION}"
```

Comment:
luajit2 needs to be imported properly as a submodule.

Reply:
Done